aana/tests/deployments/test_whisper_deployment.py

```diff
@@ -67,6 +78,14 @@ async def test_transcribe(self, setup_deployment, audio_file):
         output = await handle.transcribe(
             audio=audio, params=WhisperParams(word_timestamps=True, temperature=0.0)
         )
+        # import json
```

Comment:
We have a special env variable for that now: https://mobiusml.github.io/aana_sdk/pages/testing/#save_expected_output-environment-variable

So please use it instead and remove the commented out code.

Reply:
Removed.